### DISCUSSION TITLE
Calculos de PVs - Elfo Arcanista

### DISCUSSION THREAD

**Discussion by PauloHenriqueOC** [YuriAlessandro/gerador-ficha-tormenta20#303] [Problemas]
Olá, primeiramente parabéns pela iniciativa... 
achei muito intuitivo e legal a plataforma,  com certeza vai ajudar em muitas ocasiões!

Eu gostaria de reportar aqui uma inconsistencia que ocorreu a gerar uma ficha.
Gerei uma ficha de Elfo Arcanista lvl 20 e os PVs ficaram errados... no passo-a-passo ele parece considerar o HP/Nivel mas não atualiza os total de PVs...

![image](https://user-images.githubusercontent.com/24553304/135506839-795f42a0-4ce2-4e83-80b2-20d30d33ffd0.png)

**Comment by YuriAlessandro** [comment]
Bom, esse é um "problema" bem curioso. Observe: Esse mago tem 6 de constituição (que azar!) e portanto ele tem -2 de modificador

Segundo as regras:

![image](https://user-images.githubusercontent.com/14967048/135519250-30fc0a5d-41e2-4e83-ac41-c3c1401851dc.png)

Arcanista vai ter 2PV + mod Con

O código interpretou isso como 2 - 2 = 0 PV por nível (que é o que o passo-a-passo mostra)

Agora, se isso de receber 0 PV por nível é regra, eu ainda não sei. Vou tentar buscar aqui no livro como que funciona para esses casos (se tem valor mínimo de PV por nível, se pode ser assim, etc). Se você souber algo, já me diz aqui também

**Reply by PauloHenriqueOC** [reply]
Pelo que vejo o próprio manual de Tormenta20 parece prever esse cenário. Na pg. 35 onde começa a explicação sobre os pontos de vida das classe, diz que vc soma sim seu mod. de constituição mas mesmo que for um mod. negativo vc ganha no minimo 1PV por nível.

![image](https://user-images.githubusercontent.com/24553304/135521436-ca3b56b5-d184-48bc-b15a-79a366790ffe.png)

**Reply by YuriAlessandro** [reply]
Então é isso, sem mistério. Vou preparar uma solução e quando estiver no ar, eu aviso aqui. Obrigado!

**Comment by YuriAlessandro** [comment]
Problema corrigido na versão 2.2
